src/components/SelectComponents/Combobox/__tests__/Combobox.spec.tsx

```diff
@@ -17,178 +19,209 @@ describe('Combobox', () => {
+                };
+                rerenderView(newProps);
+                await setFocus();
+
+                expect(view.asFragment()).toMatchSnapshot();
+                expect(screen.getByText(SUGGESTIONS[1].name)).toBeInTheDocument();
             });
         });
         describe('when focused', () => {
-            it('should add additional attributes to input field when component is focused', () => {
-                wrapper.setProps({ inputAttrs: { 'data-test': true, title: 'some title' } });
-                wrapper.update();
-                setFocusOnInput();
-
-                expect(wrapper.find('input').html()).toMatch('data-test="true"');
-                expect(wrapper.find('input').html()).toMatch('title="some title"');
+            it('should add additional attributes to input field when component is focused', async () => {
+                const newProps = {
+                    inputAttrs: { 'data-test': true, title: 'some title' },
+                };
+                rerenderView(newProps);
+                await setFocus();
+                const inputField = screen.getAllByRole('textbox')[0];
+
+                expect(inputField.outerHTML).toMatch('data-test="true"');
+                expect(inputField.outerHTML).toMatch('title="some title"');
             });
-            it('should render selection placeholder', () => {
-                setFocusOnInput();
-                expect(wrapper.find('input').props().placeholder).toEqual(inputPlaceholder);
+            it('should render selection placeholder', async () => {
+                await setFocus();
+                const inputField = screen.getAllByRole('textbox')[0];
+
+                expect(inputField.getAttribute('placeholder')).toEqual(inputPlaceholder);
             });
-            it('should show the selected value as placeholder if available', () => {
-                setFocusOnInput();
-                expect(wrapper.find('input').props().placeholder).toEqual(inputPlaceholder);
-
-                wrapper.setProps({ selectedSuggestion: SUGGESTIONS[1] });
-                wrapper.update();
-                setFocusOnInput();
-                expect(wrapper.find('input').props().placeholder).toEqual(SUGGESTIONS[1].name);
+            it('should show the selected value as placeholder if available', async () => {
+                await setFocus();
+                const inputField = screen.getAllByRole('textbox')[0];
+
+                expect(inputField.getAttribute('placeholder')).toEqual(inputPlaceholder);
+                const newProps = {
+                    suggestions,
+                    suggestionToString,
+                    inputPlaceholder,
+                    noSuggestionsPlaceholder,
+                    onSelectionAdd: mockOnSelectionAdd,
+                    onInputValueChange: mockOnInputValueChange,
+                    onBlur: mockOnBlur,
+                    selectedSuggestion: SUGGESTIONS[1],
+                };
```

Comment:
are all these props new and needed to be listed here?

Reply:
:+1:

Reply:
Fixed